src/clj/game/core/flags.clj

```diff
@@ -251,18 +251,37 @@
   [card]
   (or (not (rezzed? card)) (not (:cannot-host (card-def card)))))
 
+(defn agenda? [card]
```

Comment:
Great additions!

Reply:
Thank you! I want to go through and "convert" all of the existing type checks like this to the same, but that's much bigger than this batch of fixes.